cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -21,6 +21,27 @@ extensions:
       some_storage:
         elasticsearch:
           index_prefix: "jaeger-main"
```

Comment:
this should no longer be valid

Reply:
actually, it still works, as a global coverage value, I did it right here https://github.com/jaegertracing/jaeger/blob/f2e1416adeccd2da6affe8d9fb2e8e3ad61031cf/plugin/storage/es/options.go#L347